src/plugins/vis_type_vislib/public/vislib/partials/touchdown_template.tsx

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import React from 'react';
+import ReactDOM from 'react-dom/server';
+
+interface Props {
+  wholeBucket: boolean;
+}
+
+export const touchdownTemplate = ({ wholeBucket }: Props) => {
+  return ReactDOM.renderToStaticMarkup(
+    <p className="visTooltip__header">
+      <i className="fa fa-info-circle visTooltip__headerIcon" />
+      <span className="visTooltip__headerText">
+        {wholeBucket ? 'Part of this bucket' : 'This area'}
```

Comment:
We need an extra space here because it is rendered as
![image](https://user-images.githubusercontent.com/17003240/118606547-3b370f00-b7c0-11eb-82f1-766877dc65c2.png)

Reply:
Ah good catch! Updated in [`11ff548` (#100207)](https://github.com/elastic/kibana/pull/100207/commits/11ff548bdf19b82201f6593e0e35d9a82b47a8fa)